### ISSUE TITLE
`after_stat` no longer able to be imported

### ISSUE THREAD

**Issue by conleyst** [has2k1/plotnine#495]
It seems that `after_stat` is not longer able to be imported. For reference, I'm using `v0.7.1`. This can be recreated with,

```
from plotnine import *
import pandas as pd

df = pd.DataFrame({
    'var1': list('abbcccddddeeeee'),
    'cat': list('RSRSRSRRRSRSSRS')
})

(
    ggplot(df, aes('var1')) +
    geom_bar() + 
    geom_text(aes(label=after_stat('count')), stat='count')
)
```

**Comment by AndreasMadsen** [comment]
This appears to be fixed in the development version (checked fc90653e4b3e18d43f153e5a7e1d9d5cfce5a6fc).

**Comment by has2k1** [comment]
`after_stat` is part of the new API for evaluating aesthetics. It will ship in the next release. Sometime back I made an error and regenerated the documentation for `stable` and it pulled in examples with features that had not yet shipped. I have fixed it.
